src/core_plugins/kibana/index.js

```diff
@@ -39,10 +39,27 @@ module.exports = function (kibana) {
         ],
 
         injectVars: function (server, options) {
-          const config = server.config();
+          let serverConfig = server.config();
+
+          //DEPRECATED SETTINGS
+          //: is URL the default? if setting to undefined has no effect, it means it reverted back to the default
+          //keeping this logic for backward compatibilty.
+          const configuredUrl = server.config().get('tilemap.url');
```

Comment:
ok, that would require removing the defaults from the settings. I wasn't sure if we wanted to do that altogether in a minor release. Also, if the proxy breaks for whatever reason, we won't have the fallback.